tests/testthat/test-code-of-conduct.R

```diff
@@ -1,5 +1,5 @@
 test_that("use_code_of_conduct() creates promised file", {
   create_local_project()
-  use_code_of_conduct()
+  use_code_of_conduct("test@gmail.com")
```

Comment:
Nitpick: I try to use an `@example.com` address in these settings because I once got RFC 2606 and RFC 6761 angrily quoted at me for doing otherwise. (But, unlike my previous fiasco, this is unlikely to result in anyone sending email to `test@gmail.com`. Like, LOTS of email 😬.)